Another clean up for global.mtls.enable (#6611)
istio/istio.io: content/en/docs/setup/install/istioctl/index.md

for establishing a production environment, unlike the larger `demo` profile that
is intended for evaluating a broad set of Istio features.

> If you want to enable `grafana` dashboard on top of the `default` profile, you can set the `grafana` related configuration parameters:

Reviewer: ```suggestion
To enable the Grafana dashboard on top of the `default` profile, set the `addonComponents.grafana.enabled` configuration parameter with the following command:
```

{{< text bash >}}

> $ istioctl manifest apply --set values.grafana.enabled=true

Reviewer 2: recommended way to enable grafana: `istioctl manifest apply --set addonComponents.grafana.enabled=true`

Author: done
